plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
+
+/**
+ * Scripts & Styles.
+ *
+ * @param  [type]    $hook
```

Comment:
`[type]` should be defined. There should also be a description explaining what `$hook` is.